### PR TITLE
Fix for https://github.com/strongloop/loopback/issues/1401

### PR DESCRIPTION
**Pull request by raymondfeng**
/to @ritch 

Connect to https://github.com/strongloop/loopback/issues/1401

### CODE REVIEW

**Comment by raymondfeng** [comment]
@ritch ping

File: test/include.test.js [hunk 1]
```diff
@@ -690,3 +693,96 @@ function clearAndCreate(model, data, callback) {
...
+        }
+      }
+    });
+
+    async.waterfall([
+      createChallengers,
+      createGameParticipations,
+      createResults,
+      function() {
+        done();
```
Comment (ritch) [inline test/include.test.js]: ignoring errors here on purpose?

**Comment by ritch** [comment]
Minor comment otherwise LGTM

File: test/include.test.js (hunk 1 above)
Comment (raymondfeng) [inline test/include.test.js]: Good catch, fixed.

**Pull request merged**